### ISSUE TITLE
Project or codebase?

### ISSUE THREAD

**Issue by Ainali** [publiccodenet/standard#409]
The fourth requirement in [Make contributing easy](https://standard.publiccode.net/criteria/make-contributing-easy.html) says:

> The project MUST have communication channels for users and developers, for example email lists.

However, previous requirements talk about that *the codebase* need an email and issue tracker. Is this an omission and we should change `project` to `codebase` or is it done on purpose?

**Comment by ericherman** [comment]
Yes, should be "codebase" (or "community"), not "project"

**Comment by ericherman** [comment]
```
$ git grep -il project criteria
criteria/continuous-integration.md
criteria/document-maturity.md
criteria/document-objectives.md
criteria/open-to-contributions.md
criteria/require-review.md
criteria/reusable-and-portable-codebases.md
criteria/understandable-english-first.md
```

These are mostly not in requirements (except one use of "project or codebase"), but rather in the additional information.

There is room for improvement in these areas also, yet some uses of "project" are still appropriate.

**Comment by Ainali** [comment]
Oh. Yes indeed. I'll make another PR with a suggestion for what to change.

**Comment by Ainali** [comment]
Fixed by #410 and #411
